3.3-apache/Dockerfile

```diff
@@ -28,11 +44,12 @@ RUN apt-get update \
  && sed -i /etc/modsecurity.d/modsecurity.conf \
       -e 's/SecRuleEngine DetectionOnly/SecRuleEngine On/g'
 
-COPY httpd-logging-before-modsec.conf /usr/local/apache2/conf/extra/
-COPY httpd-logging-after-modsec.conf  /usr/local/apache2/conf/extra/
-COPY docker-entrypoint.sh /
+COPY common/modsecurity.conf /etc/modsecurity.d/
+COPY 3.3-apache/conf/*.conf /usr/local/apache2/conf/extra/
+COPY 3.3-apache/docker-entrypoint.sh /
+COPY common/modsec-activate-rules.sh /
 
-EXPOSE 80
+EXPOSE ${PORT}
```

Comment:
I'm not sure this will work. The EXPOSE directive is basically just metadata of the image and as far as I know it's set at build time. So it'll be always `80` anyway.

Reply:
We tested it and it is as expected always `80`, so might as well hard-code it to to cause confusion.

Reply:
It _does_ work, but only at build time. So whatever the `ENV` directive inside the Dockerfile sets `$PORT` to...